src/Database/Attach/File.php

```diff
@@ -270,33 +258,34 @@ public function getSizeAttribute()
     /**
      * Outputs the raw file contents.
      *
-     * @param string $disposition The Content-Disposition to set, defaults to inline
-     * @param bool $returnResponse Defaults to false, returns a Response object instead of directly outputting to the browser
-     * @return Response | void
+     * @param string $disposition The Content-Disposition to set, defaults to `inline`
+     * @param bool $returnResponse Defaults to `false`, returns a Response object instead of directly outputting to the
+     *  browser
+     * @return \Illuminate\Http\Response|void
      */
-    public function output($disposition = 'inline', $returnResponse = false)
+    public function output(string $disposition = 'inline', bool $returnResponse = false)
     {
         $response = response($this->getContents())->withHeaders([
             'Content-type'        => $this->getContentType(),
             'Content-Disposition' => $disposition . '; filename="' . $this->file_name . '"',
-            'Cache-Control'       => 'private, no-store, no-cache, must-revalidate, pre-check=0, post-check=0, max-age=0',
+            'Cache-Control'       => 'private, no-store, no-cache, must-revalidate, max-age=0',
```

Comment:
What's the reason for this change?

Reply:
`pre-check` and `post-check` are invalid Cache Control attributes. Internet Explorer 5 introduced them, and they haven't been used for years. [Relevant blog post for more info](https://docs.microsoft.com/en-us/archive/blogs/ieinternals/internet-explorers-cache-control-extensions?_fsi=f81xrzF2)